docs/en/docs/deployment/manually.md

```diff
@@ -131,7 +131,7 @@ Now you can use Trio internally in your app. Or even better, you can use AnyIO,
 
 ## Deployment Concepts
 
-These examples run the server program (e.g Uvicorn), starting **a single process**, listening on all the IPs (`0.0.0.0`) on a predefined port (e.g. `80`).
+These examples run the server program (ex. Uvicorn), starting **a single process**, listening on all the IPs (`0.0.0.0`) on a predefined port (ex. `80`).
```

Comment:
I think `e.g.` was more accurate.
`ex.` may be used to mean `execpt`, `exercise` and is sometimes used for `for example`....
ref: http://www.jvhighschool.com/study-tip-how-to-use-i-e-e-g-and-ex-properly/